<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trainingMode</class>
 <widget class="QDialog" name="trainingMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labelWord">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>160</y>
     <width>375</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 51, 0);
background-color: rgba(0, 0, 127, 150);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>press it --&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCheck">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>375</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127, 150);
color: rgba(0, 210, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheck">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>135</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(0, 0, 127, 100); color: rgb(85, 255, 0)}
QPushButton::hover{background-color: rgba(0, 0, 127, 150)}</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRepeat">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>115</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(0, 0, 127, 100); color: rgb(204, 204, 255)}
QPushButton::hover{background-color: rgba(0, 0, 127, 150)}</string>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIDontKnow">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>290</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(0, 0, 127, 100); color: rgb(0, 255, 204)}
QPushButton::hover{background-color: rgba(0, 0, 127, 150)}</string>
   </property>
   <property name="text">
    <string>Show Translation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 210, 255);</string>
   </property>
   <property name="text">
    <string>Russian only</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEng">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 210, 255);</string>
   </property>
   <property name="text">
    <string>English only</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEndAndRus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 210, 255);</string>
   </property>
   <property name="text">
    <string>English + Russian</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelCountWords">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>20</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(255, 255, 0);
background-color: rgba(255, 0, 0, 100);}
QToolTip{border: 0px; background-color: rgb(0, 85, 127); font: 11pt Comic Sans MS; color: rgb(255,255,0);qproperty-alignment: AlignCenter;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSkip">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>115</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(0, 0, 127, 100); color: rgb(204, 204, 255)}
QPushButton::hover{background-color: rgba(0, 0, 127, 150)}</string>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStartOver">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(255, 0, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Start Over</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(255, 0, 0, 100); color: rgb(255, 255, 0)}
QPushButton::hover{background-color: rgba(255, 0, 0, 150)}</string>
   </property>
   <property name="text">
    <string>Set Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCountText">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>20</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(255, 255, 0);
background-color: rgba(255, 0, 0, 100);}
QToolTip{border: 0px; background-color: rgb(0, 85, 127); font: 11pt Comic Sans MS; color: rgb(255,255,0);qproperty-alignment: AlignCenter;}</string>
   </property>
   <property name="text">
    <string>You've Translated: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEntireNumber">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>20</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(255, 255, 0);
background-color: rgba(255, 0, 0, 100);}
QToolTip{border: 0px; background-color: rgb(0, 85, 127); font: 11pt Comic Sans MS; color: rgb(255,255,0);qproperty-alignment: AlignCenter;}</string>
   </property>
   <property name="text">
    <string>/0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 0, 0, 100);
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Time: 00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
